storage/preferences/src/main/java/de/danoeh/antennapod/storage/preferences/UserPreferences.java

```diff
@@ -231,6 +235,49 @@ public static boolean showSkipOnCompactNotification() {
         return showButtonOnCompactNotification(NOTIFICATION_BUTTON_SKIP);
     }
 
+    public static boolean showNextChapterOnCompactNotification() {
+        return showButtonOnCompactNotification(NOTIFICATION_BUTTON_NEXT_CHAPTER);
+    }
+
+    public static List<Integer> getFullNotificationButtons() {
+        String[] buttons = TextUtils.split(
+            prefs.getString(PREF_FULL_NOTIFICATION_BUTTONS,
+                    NOTIFICATION_BUTTON_SKIP + ","
+                        + NOTIFICATION_BUTTON_PLAYBACK_SPEED),
+            ",");
+
+        List<Integer> notificationButtons = new ArrayList<>();
+        for (String button : buttons) {
+            notificationButtons.add(Integer.parseInt(button));
+        }
+        return notificationButtons;
+    }
+
+    /**
+     * Helper function to return whether the specified button should be shown on full
+     * notifications.
+     *
+     * @param buttonId Either NOTIFICATION_BUTTON_REWIND, NOTIFICATION_BUTTON_FAST_FORWARD,
+     *                 NOTIFICATION_BUTTON_SKIP, NOTIFICATION_BUTTON_PLAYBACK_SPEED
+     *                 or NOTIFICATION_BUTTON_NEXT_CHAPTER.
+     * @return {@code true} if button should be shown, {@code false}  otherwise
+     */
+    private static boolean showButtonOnFullNotification(int buttonId) {
+        return getFullNotificationButtons().contains(buttonId);
+    }
+
+    public static boolean showSkipOnFullNotification() {
+        return showButtonOnFullNotification(NOTIFICATION_BUTTON_SKIP);
+    }
+
+    public static boolean showNextChapterOnFullNotification() {
+        return showButtonOnFullNotification(NOTIFICATION_BUTTON_NEXT_CHAPTER);
+    }
+
+    public static boolean showPlaybackSpeedOnFullNotification() {
+        return showButtonOnFullNotification(NOTIFICATION_BUTTON_PLAYBACK_SPEED);
+    }
```

Comment:
Hmm, this feels like quite a bit of code for something that we want to remove later (see #6650). I think it would make sense to remove the setting here already, so that you don't need to add code than will then later be deleted anyway. That would also make the settings UI a lot easier because you no longer need the minimum/maximum number of selected items